pom.xml

```diff
@@ -141,6 +141,11 @@
                 <artifactId>kubernetes-httpclient-jdk</artifactId>
                 <version>${fabric8.version}</version>
             </dependency>
+            <dependency>
```

Comment:
Presumably this is needed because Java Operator SDK is still on an older version? Can you add a comment that this can be removed when we move to Java Operator SDK 4.9.2? (As I see in the main branch they've just updated this dependency so it should be in that release)

Reply:
Yeah sure, good point, thanks :)

Reply:
Mainly one test failed because of that:

```
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.378 s <<< FAILURE! - in io.strimzi.kafka.access.KafkaAccessReconcilerTest
[ERROR] io.strimzi.kafka.access.KafkaAccessReconcilerTest  Time elapsed: 0.378 s  <<< ERROR!
java.lang.NoClassDefFoundError: io/fabric8/kubernetes/client/V1Alpha1DynamicResourceAllocationAPIGroupDSL
Caused by: java.lang.ClassNotFoundException: io.fabric8.kubernetes.client.V1Alpha1DynamicResourceAllocationAPIGroupDSL
```